rlp/src/main/scala/io/iohk/ethereum/rlp/RLPImplicits.scala

```diff
@@ -51,7 +50,7 @@ object RLPImplicits {
   implicit val bigIntEncDec = new RLPEncoder[BigInt] with RLPDecoder[BigInt] {
 
     override def encode(obj: BigInt): RLPValue = RLPValue(
-      if (obj.equals(BigInt(0))) byteToByteArray(0: Byte) else obj.toUnsignedByteArray
+      if (obj.equals(BigInt(0))) byteToByteArray(0: Byte) else bigIntToUnsignedByteArray(obj)
```

Comment:
It looks like `toUnsignedByteArray` in BigIntExtensionMethods.BigIntAsUnsigned
has the same implementation as: `bigIntToUnsignedByteArray` in RLP

Is his neccessary?

Reply:
It is indeed a carbon copy. I brought it over because this was the only thing it used from the utils, and I didn't want to create a "mantis-utils" library for RLP to depend on, and it also didn't look like the entire `ByteUtils` should be be moved to RLP. But I'm open to suggestions.

Reply:
Maybe a dedicated `mantis-bytes` library?

Reply:
Created a `bytes` project, moved `bigIntToUnsignedByteArray` to `ByteUtils`, and now `BigIntExtensionMethods` calls that as well.

Reply:
Did the same treatment for crypto.